internal/server/singleprocess/state/application_test.go

```diff
@@ -2,13 +2,15 @@ package state
 
 import (
 	"testing"
+	"time"
 
 	"github.com/stretchr/testify/require"
 	/*
 		"google.golang.org/grpc/codes"
 		"google.golang.org/grpc/status"
 	*/
```

Comment:
This isn't you, but this is weird and we should prob delete it.

Reply:
Good call! Very strange, I'll remove 👍🏻